examples/angular-cli/src/stories/addon-actions.stories.ts

```diff
@@ -1,23 +1,34 @@
-import { storiesOf } from '@storybook/angular';
 import { action } from '@storybook/addon-actions';
 import { Button } from '@storybook/angular/demo';
 
-storiesOf('Addon|Actions', module)
-  .add('Action only', () => ({
-    component: Button,
-    props: {
-      text: 'Action only',
-      onClick: action('log 1'),
-    },
-  }))
-  .add('Action and method', () => ({
-    component: Button,
-    props: {
-      text: 'Action and Method',
-      onClick: e => {
-        console.log(e);
-        e.preventDefault();
-        action('log2')(e.target);
-      },
+export default {
+  title: 'Addon|Actions',
+};
+
+export const actionOnly = () => ({
+  component: Button,
+  props: {
+    text: 'Action only',
+    onClick: action('log 1'),
+  },
+});
+
+actionOnly.story = {
```

Comment:
Is there a way to type this? In `strict: true` you most likely have to 
`export const actionOnly: any` in order to call `.story`

Maybe `actionOnly: Story = () => { ... }`?

Reply:
@kroeder great idea!

https://github.com/storybookjs/storybook/issues/7677